specifications/api/swagger.yaml

```diff
@@ -1410,6 +1410,30 @@ components:
     ApiNetworkClock: &ApiNetworkClock
       <<: *networkInformationNtpStatus
 
+    nullableEpochInfo: &nullableEpochInfo
+      <<: *epochInfo
+      nullable: True
+
+    ApiEraInfo: &ApiEraInfo
+      type: object
+      required:
+        - byron
+        - shelley
+        - allegra
+        - mary
+      properties:
+        byron: *nullableEpochInfo
+        shelley: *nullableEpochInfo
+        allegra: *nullableEpochInfo
+        mary: *nullableEpochInfo
+      description: |
+
+        If and when each era started or will start.
+
+        The object is keyed by era names. The values either describe the epoch boundary
+        when the era starts (can be in the future or in the past), or are null if not yet
+        confirmed on-chain.
```

Comment:
Because of
> (can be in the future or in the past)

wouldn't it make sense to provide a `era` field either here, or in `/network/information`?

A non-null `mary` does not necessarily mean that we're in Mary. We could also be just before Mary.

Reply:
I thought it would go like this : we know something like mary will happen (null), then we know when it happens (by specifying era it starts), and we also now current slot/era. So everything is fine;-)